data/games.py

```diff
@@ -0,0 +1,37 @@
+def steam550(config):
+    config.vName = "L4D2"
+    config.vMin = 4
+    config.vMax = 8
+    config.vTeams = [4, 4]
+
+
+def steam730(config):
+    config.vName = "CS:GO"
+    config.vMax = 20
+
+
+def steam381210(config):
+    config.vName = "Dead by Daylight"
+    config.vMin = 4
+    config.vMax = 5
+    config.vTeams = [1, 4]
+
+
+def steam548430(config):
+    config.vName = "Deep Rock Galactic"
+    config.vMin = 4
+    config.vMax = 4
+
+
+def steam945360(config):
+    config.vName = "Among Us"
+    config.vMin = 4
+    config.vMax = 16
+    config.vTeams = [4, 4]
+
+
+def steam976730(config):
+    config.vName = "Halo: MCC"
+    config.vMin = 4
+    config.vMax = 16
+    config.vTeams = [4, 4]
```

Comment:
Outside of the scope of this PR, but it might be worth looking into launching games outside of Steam (e.g. Xbox Game Pass)

Reply:
This was the reason for the 'steam' namespacing and the decorator definition. `@game(steam: 550)`, `@game(blizzard: xyz)`, and `@game(custom_foo)` are all possibilities.